addons/purchase/models/purchase.py

```diff
@@ -1151,6 +1161,38 @@ def _onchange_quantity(self):
 
         self.price_unit = price_unit
 
+    @api.onchange('product_packaging_id', 'product_qty', 'product_uom')
+    def _onchange_product_packaging(self):
+        if self.product_packaging_id and self.product_qty:
+            packaging_uom = self.product_packaging_id.product_uom_id
```

Comment:
:+1:

Reply:
@amoyaux I reversed this change ;)
Since now we can suggest packaging according to the product_qty. I always find a suitable packaging first, if no, just set remove packaging instead of give a warning. the warning will only be raise, if user force to use a wrong one. 
what do you think?